src/components/shared/NavBar.tsx

```diff
@@ -108,66 +107,85 @@ const NavBar: FC<NavBarProps> = ({}) => {
-            `} onClick={handleLanguageClick}>
+            `}
+            onClick={handleLanguageClick}
+          >
             <FaGlobe
-               className={`${
+              className={`${
                 !isButtonClicked ? "text-secondary-500" : "text-white"
-                } ${isDarkMode ?  "text-white" : "text-secondary-500" }  z-4 object-contain py-1 w-[29px] text-[32px]`}
+              } ${
+                isDarkMode ? "text-white" : "text-secondary-500"
+              }  z-4 object-contain py-1 w-[29px] text-[32px]`}
             />
             {showLanguages && (
-            <div ref={languageRef} className="overflow-hidden absolute z-50 bottom-[-170px] bg-white border-[1px] border-gray-200 rounded-lg shadow-md flex flex-col items-center">          
-            {/* map languages */}
-            {Object.entries(languages).map(([languageName, languageCode]) => (
-                <button
-                  key={languageCode}
-                  onClick={() => handleLanguageChange(languageCode)}
-                  className={`hover:bg-gray-300 px-5 py-2 w-full ${selectedLanguage === languageCode ? 'bg-secondary-500 py-3 font-bold text-white' : ''}`}
-                >
-                  {t(languageName)}
-                </button>
-              ))}
-              {/* Add more buttons for additional languages */}
-            </div>
-            )} 
+              <div
+                ref={languageRef}
+                className="overflow-hidden absolute z-50 bottom-[-170px] bg-white border-[1px] border-gray-200 rounded-lg shadow-md flex flex-col items-center"
+              >
+                {/* map languages */}
+                {Object.entries(languages).map(
+                  ([languageName, languageCode]) => (
+                    <button
+                      key={languageCode}
+                      onClick={() => handleLanguageChange(languageCode)}
+                      className={` px-5 py-2 w-full 
+                      ${
+                        selectedLanguage === languageCode
+                          ? "bg-secondary-500 py-3 font-bold text-white"
+                          : "hover:bg-gray-300 bg-white-100 text-black bg-[#F5F5F5]"
+                      }`}
+                    >
+                      {t(languageName)}
+                    </button>
+                  )
+                )}
+                {/* Add more buttons for additional languages */}
+              </div>
+            )}
           </div>
-      
-            {/* sign up button */}
+
+          {/* sign up button */}
           <Button
             type="button"
             className={`max-md:px-4 max-md:py-2 md:mr-4 px-7 py-2 rounded-[3px] text-black border border-[#000] ${
-              isDarkMode ? "border border-[#F5F5F5] text-white" : "border border-[#000] text-black"
+              isDarkMode
+                ? "border border-[#F5F5F5] text-white"
+                : "border border-[#000] text-black"
```

Comment:
for this icon dark mode and translation button, I removed the border and hover effect and the bg you can check on deployed version so no need anymore

Reply:
which branch?